packages/opencensus-core/src/metrics/metric-registry.ts

```diff
@@ -165,6 +166,68 @@ export class MetricRegistry {
     return derivedDoubleGauge;
   }
 
+  /**
+   * Builds a new Int64 cumulative to be added to the registry. This API is
+   * useful when you want to manually increase and reset values as per service
+   * requirements.
+   *
+   * @param {string} name The name of the metric.
```

Comment:
What would you think about removing these types in the JSDoc that are the same as the TS types?

Reply:
sure, done.